src/com/library/Entities/Category.java

```diff
@@ -0,0 +1,38 @@
+package com.library.Entities;
+
+public class Category {
```

Comment:
[Fix] eu vad aceasta clasa ca fiind un enum, nu vad ce valoare ar aduce daca ar avea si id si am salva in baza de date